src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```